UPGRADE.md

```diff
@@ -1,6 +1,17 @@
 eZ Platform Site API upgrade instructions
 =========================================
 
+Upgrading from 2.3 to 2.4
+-------------------------
+
+Controllers that extend from `Netgen\Bundle\EzPlatformSiteApiBundle\Controller\Controller` and are registered inside dependency injection container must set `parent` config key within service definition, example:
```

Comment:
This is because of `setContainer` and `setSite` calls, right? In that case it's not really a must, I would mention required calls explicitly, and then suggest that it can be done by setting parent configuration which already does it.

Otherwise this looks good :)